distributed/core.py

```diff
@@ -1169,34 +1169,36 @@ def error_message(e, status="error"):
     MAX_ERROR_LEN = dask.config.get("distributed.admin.max-error-length")
     tblib.pickling_support.install(e, *collect_causes(e))
     tb = get_traceback()
-    e2 = truncate_exception(e, MAX_ERROR_LEN)
+    tb_text = "".join(traceback.format_tb(tb))
+    e = truncate_exception(e, MAX_ERROR_LEN)
     try:
-        e3 = protocol.pickle.dumps(e2, protocol=4)
```

Comment:
Do you know why protocol 4 was used earlier ?

Reply:
I believe it is some legacy code: https://github.com/dask/distributed/pull/4699#discussion_r621440799